### ISSUE TITLE
Medibots heal slimpeople to death

### ISSUE THREAD

**Issue by Anonmare** [tgstation/tgstation#46736]
## Round ID: 
120047

## Testmerges:
None

## Reproduction:
Make medibot
Put near slimeperson (stargazer in my example)
Watch it remove all their "blood"

**Comment by Ty-the-Smonk** [comment]
This has been a thing for ages, Medibots are after all the apex predator of slime people

**Comment by Tharcoonvagh** [comment]
But now they don't inject chems, only tend wounds

**Comment by Anonmare** [comment]
I'm not sure what it is but I watched one heal a slimeperson to death and leave the humans alone and I definitely no it wasn't emagged

**Comment by Ty-the-Smonk** [comment]
Uhhhh does tend wounds or experimental heal toxic? Is so that’s why

**Comment by Anonmare** [comment]
Medibots have a pseudo tend wounds that heals tox and oxy

**Comment by PKPenguin321** [comment]
I'm tempted to call this a non-issue, emergent gameplay thing.

**Comment by kriskog** [comment]
@ExcessiveUseOfCobblestone It was changed so only the toxin specific medibot now heals toxins, right?

**Comment by ExcessiveUseOfCobblestone** [comment]
yes, although this probably still happens with the toxin-healing medbot.

I call that a feature 😏

**Comment by kriskog** [comment]
Thats what I was thinking too
